guides/v2.3/graphql/mutations/create-customer-address.md

```diff
@@ -95,6 +95,17 @@ The `createCustomerAddress` mutation returns the following attributes:
 
 {% include graphql/customer-address-output.md %}
 
+## Errors
+
+Error | Description
+--- | ---
+`Expected type CustomerAddressInput!, found "".` | The `input` attribute contains an empty value.
+`"input" value should be specified` | The `input` attribute is set but contains an empty data.
+`Required parameters are missing: firstname` | The `input.firstname` attribute was omitted or contains an empty value.
+`"Street Address" cannot contain more than 2 lines.` | The `input.street` attribute contains array with more than 2 elements.
+`Syntax Error: Expected Name, found )` | The `input` attribute was omitted.
```

Comment:
Is this error related to Magento GraphQl implementation or it's just a syntax error in the query?
If the second one, I'm not sure we should describe it in the documentation since there are tons of cases on how you can make a mistake in a query with different error messages.

Reply:
`Syntax Error: Expected Name, found )` appears when the body of `createCustomerAddress` is empty:

```
mutation {
  createCustomerAddress(
    
  ) {
    city
  }
}
```

Reply:
>  If the second one, I'm not sure we should describe it in the documentation since there are tons of cases on how you can make a mistake in a query with different error messages.

Mentioned errors were covered in the API functional tests so I added them to documentation.

Thank you, @rogyar